### DISCUSSION TITLE
Migrations Keep Cloud & Local in Sync?

### DISCUSSION THREAD

**Discussion by lokireturns** [hasura/graphql-engine#7777] [General]
So I migrated from my cloud environment to develop locally using the guide: https://hasura.io/docs/latest/graphql/cloud/projects/environments.html#moving-from-hasura-cloud-to-local-development

The front end is a simple Vue JS app, I recently switched to a brand new laptop: Pulled my front end code from GitHub and connected it to my Hasura Cloud endpoint. To my surprise all data changes made locally seem to have been synced to the cloud instance also.

Is this normal behaviour? I'm struggling to find out why this happens from the docs. I assumed I would have to migrate local changes manually back to the cloud instance.

**Comment by samuela** [comment]
Are you sure that the frontend is pointed at your local Hasura instance and not prod? Also are you talking about `mutation` queries or schema changes (migrations)?

**Reply by samuela** [reply]
If you're using `hasura console` it may be inadvertently pointed at prod and not your local instance.
